notebooks/ai-ml/ImageClassificationInSpark.ipynb

```diff
@@ -0,0 +1,383 @@
+{
```

Comment:
<p>Nit: Markdown entire link is shown</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudDataproc/cloud-dataproc/pull/167/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>